Files/Helpers/ContextFlyoutItemHelper.cs

```diff
@@ -656,6 +657,37 @@ public static List<ContextMenuFlyoutItemViewModel> GetBaseItemMenuItems(BaseLayo
                     IsPrimary = true,
                 },
                 new ContextMenuFlyoutItemViewModel()
+                {
+                    Text = "BaseLayoutItemContextFlyoutExtractionOptions".GetLocalized(),
+                    Glyph = "\xF11A",
+                    ShowItem = selectedItems.Count == 1 && selectedItems.First().PrimaryItemAttribute == StorageItemTypes.File && new [] { ".zip", ".msix", ".msixbundle" }.Contains(selectedItems.First().FileExtension.ToLowerInvariant()),
```

Comment:
Can you use `StringComparison.OrdinalIgnoreCase`?

Reply:
But that doesn't make any difference

Reply:
@duke7553 preferred for us to use this in the past, maybe he can chime in if he had a reason.